client/src/Components/StatsPage.js

```diff
@@ -66,8 +66,10 @@ class StatsProvider extends Component {
     let weekChange = this.weekChange.bind(this)
 
     return (
-      <TopNav week={week} weeks={weeks} weekChange={weekChange}>
-        <SideNav/>
+      <TopNav>
+        <ul className="right">
+          <WeekPicker week={week} weeks={weeks} onChange={weekChange} />
+        </ul>
```

Comment:
the SideNav was constant so I'm not sure why I was passing it through. My making it constant I freed up `children` to be used for the right Nav if desired. This way if we want to use TopNav without the week picker we just don't pass any children in.